app/controllers/developer_controller.rb

```diff
@@ -13,7 +13,14 @@ def update_sites
 
   def production_log
     @log = if params[:grep].present?
-             `tail -n 10000 log/production.log | grep -E '#{params[:grep]}' -C #{params[:context]} --color=never`
+             grep_str = params[:grep].to_s
+             context_int = if params[:context].respond_to?(:to_i)
+                             params[:context].to_i
+                           else
+                             flash[:warning] = 'Coercion failure on params[:context]'
```

Comment:
I'd use a more informative error and instead say "Context was not a number" or something like that.